src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerRequestForInformation.java

```diff
@@ -283,39 +274,43 @@ private void areEmailsValid(CaseData caseData, List<String> errors) {
         }
     }
 
-    private void isApplicantOnline(CaseData caseData, Applicant applicant, List<String> errors) {
-        if (applicant.isApplicantOffline()) {
-            errors.add(getErrorString(NOT_ONLINE_ERROR, caseData, applicant));
-            errors.add(USE_CREATE_GENERAL_LETTER_FOR_OFFLINE_PARTIES_ERROR);
+    private boolean isApplicantAddressInvalid(Applicant applicant) {
+        return isEmpty(applicant.getCorrespondenceAddressWithoutConfidentialCheck()) && applicant.isApplicantOffline();
```

Comment:
Reordering this conditional will avoid calling those methods if applicant is offline

Reply:
If applicant is *online* I think - but yes agreed - done.

Reply:
Yes sorry when applicant is online 😃